taskvine/src/manager/vine_file_replica_table.c

```diff
@@ -83,7 +83,7 @@ struct vine_worker_info **vine_file_replica_table_find_replication_targets(
 	{
 		if (found == q->temp_replica_count)
 			break;
-		if (skip_workers--)
+		if (skip_workers-- > 0)
```

Comment:
Let's do:
```
skip_workers--;
if (skip_workers >= 0)
```
so people don't have to worry about pre and post increment rules (assuming that the above is what you meant).